frontend/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```

Comment:
You could probably remove some of this css since we remove the logo and a few other elements